Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -13276,6 +13225,43 @@ Ref<WebBackForwardList> WebPageProxy::protectedBackForwardList() const
     return m_backForwardList;
 }
 
+template<typename F>
+void WebPageProxy::sendToWebPage(const std::optional<FrameIdentifier>& frameID, F&& sendFunction)
+{
+    if (frameID) {
+        if (RefPtr frame = WebFrameProxy::webFrame(*frameID)) {
+            if (RefPtr remotePageProxy = frame->remotePageProxy()) {
+                sendFunction(remotePageProxy.get());
+                return;
+            }
+        } else {
+            ASSERT_NOT_REACHED();
+            return;
```

Comment:
I don't think we would want to return in this case.

Reply:
Returning here matches existing behavior, but I agree maybe we shouldn't. I'll remove it.